packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -158,10 +160,13 @@ class TextTreeConfiguration {
   /// [singleLineTextConfiguration].
   final String lineBreak;
 
+  /// Whether to line break properties or to leave all properties on one line.
```

Comment:
clarify the docs by answering: Does this mean "put line breaks between properties" or "wrap properties with line breaks"?